src/ocm/components/clusterConfiguration/networkConfiguration/networkConfigurationValidation.ts

```diff
@@ -53,14 +51,9 @@ export const getNetworkInitialValues = (
   };
 };
 
-export const getNetworkConfigurationValidationSchema = (
-  initialValues: NetworkConfigurationValues,
-  hostSubnets: HostSubnets,
-) =>
+export const getNetworkConfigurationValidationSchema = () =>
   Yup.lazy<NetworkConfigurationValues>((values) =>
     Yup.object<NetworkConfigurationValues>().shape({
-      apiVip: vipValidationSchema(hostSubnets, values, initialValues.apiVip),
```

Comment:
I may not have expressed this correctly. We should only remove the validation that enforces apiVip / ingressVip being set when DHCP is off.

The rest of the validations (they must be unique, in the range of the cluster network etc) must remain the same. If they fail it would be when the user is editing them, so we don't want them to provide an invalid value (that most likely would be rejected by the API on a patch)

Reply:
I miss understood. Now I fix it